CMakeLists.txt

```diff
@@ -59,6 +59,11 @@ include(make/policies.cmake NO_POLICY_SCOPE)
 # this must be done prior to the project() command and the var
 # must be set in the cache.
 if ("${CMAKE_GENERATOR}" MATCHES "Visual Studio")
+  set(CMAKE_SYSTEM_VERSION_TEMP  $ENV{WindowsSDKVersion})
+  string(REGEX REPLACE "([0-9.]+).*" "\\1" CMAKE_SYSTEM_VERSION_TEMP "${CMAKE_SYSTEM_VERSION_TEMP}" )
+  if(CMAKE_SYSTEM_VERSION_TEMP)
```

Comment:
style: As you can see this does not match all the other `if`'s as it has no space before the `(`.